src/Factors/factors_dataframes.jl

```diff
@@ -7,7 +7,7 @@
 Convert a Factor to a DataFrame
 """
 function Base.convert(::Type{DataFrame}, ϕ::Factor)
-    df = DataFrames.DataFrame(pattern(ϕ))
+    df = DataFrames.DataFrame(pattern(ϕ), :auto)
```

Comment:
This change breaks 0.2.1 however fixes issues in 1.*, i think leaving support for 0.21 should be fine?  It had its last  release in Oct 2020 and there are a lot more releases ahead of it

Reply:
Yes, that's fine.